app/controllers/application_controller.rb

```diff
@@ -30,6 +30,10 @@ def user_for_paper_trail
     user_signed_in? ? current_user.id : 'Guest'
   end
 
+  def default_url_options
+    { host: 'http://test.host' || 'localhost' || 'https://ebwiki.org' }
```

Comment:
I would rather this be listed in each of the environments files, with the value set to localhost for dev and test environments and the domain name for production.